sda-commons-server-mongo-testing/src/main/java/org/sdase/commons/server/mongo/testing/MongoDbRule.java

```diff
@@ -71,23 +74,18 @@
     static final MongodStarter INSTANCE = getMongoStarter();
 
     private static MongodStarter getMongoStarter() {
-      de.flapdoodle.embed.process.store.ExtractedArtifactStoreBuilder artifactStoreBuilder =
-          new ExtractedArtifactStoreBuilder()
-              .defaults(Command.MongoD)
-              .download(createDownloadConfig());
-      // avoid recurring firewall requests on mac, see
-      // https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo/issues/40#issuecomment-14969151
-      if (SystemUtils.IS_OS_MAC_OSX) {
-        artifactStoreBuilder.executableNaming((prefix, postfix) -> "mongod");
```

Comment:
That doesn't seem to be necessary anymore. The default code always stores it in the same location. And I don't get firewall warnings.